types/setting.go

```diff
@@ -409,6 +412,19 @@ var (
 		},
 	}
 
+	SettingDefinitionBackupExecutionTimeout = SettingDefinition{
+		DisplayName: "Backup Execution Timeout",
+		Description: "In minutes. This setting determines the timeout for the backup engine execution. Set to 0 to disable the timeout.",
```

Comment:
Suggest rewording the `Set to 0 to disable the timeout.` as I suggested in longhorn/longhorn.

Reply:
Modified. Thanks.